### DISCUSSION TITLE
Apple Silicon support?

### DISCUSSION THREAD

**Discussion by JoeHershkie** [CarlGao4/Demucs-Gui#19] [General]
Can I configure this to support M-series SOCs?

**Comment by CarlGao4** [comment]
I have to wait for pytorch to add this support

**Reply by CarlGao4** [reply]
I'll add it to v1.0

**Comment by CarlGao4** [comment]
The first packed native ARM64 macOS application has just been released!

**Reply by chriscjcj** [reply]
I installed Demucs-GUI_1.1a2_macOS_ARM64.dmg on my MacBook Pro with M1 Pro CPU. (16GB RAM and MacOS 14.3.) It crashes on launch. I would be happy to send you the crash log, but I'd rather not post it publicly for fear that there may be sensitive info in it. 

Here's a piece of it which might be helpful.
Crashed Thread:        0  Dispatch queue: com.apple.main-thread

> Exception Type:        EXC_CRASH (SIGABRT)
> Exception Codes:       0x0000000000000000, 0x0000000000000000
> 
> Termination Reason:    Namespace SIGNAL, Code 6 Abort trap: 6
> Terminating Process:   Demucs-GUI [57940]

Also... I recently sent you $40 on PayPal. Thanks so much for all your great work on this app!!!

**Reply by CarlGao4** [reply]
Thanks for your support!
In fact, this is not enough...
You can send your log to my email

**Reply by chriscjcj** [reply]
Full crash log has been sent to your e-mail. Thank you!

**Reply by CarlGao4** [reply]
Does 1.1 crash too?

**Reply by chriscjcj** [reply]
1.1 Does not crash! And it's very fast on the M1 Pro. Thanks again for all your amazing work. 🙏🏻
